python/lsst/meas/algorithms/measureApCorr.py

```diff
@@ -175,12 +200,15 @@ def run(self, exposure, catalog):
             - flux sigma field (e.g. base_PsfFlux_fluxSigma): 2d model of error
         """
         bbox = exposure.getBBox()
+        import lsstDebug
+        display = lsstDebug.Info(__name__).display
 
         self.log.info("Measuring aperture corrections for %d flux fields" % (len(self.toCorrect),))
         # First, create a subset of the catalog that contains only selected stars
         # with non-flagged reference fluxes.
         subset1 = [record for record in self.starSelector.selectStars(exposure, catalog).starCat
-                   if not record.get(self.refFluxKeys.flag)]
+                   if not record.get(self.refFluxKeys.flag) and
+                   numpy.isfinite(record.get(self.refFluxKeys.flux))]
```

Comment:
Please indent a bit here so that it's obvious that it's not a new statement. The `if` aligned with the `numpy` confuses a quick scan of the code.